bin/sync_all

```diff
@@ -8,7 +8,7 @@ fi
 ./manage.py migrate --noinput --database bedrock
 ./manage.py rnasync
 ./manage.py cron update_ical_feeds
```

Comment:
outside the scope of this PR, but I just realized we probably need to add "--database bedrock" to the other mgmt cmds, including adding support where needed